cli/sawtooth_cli/config.py

```diff
@@ -262,6 +272,43 @@ def _accept(candidate, public_key, prefix):
         raise AssertionError('Unknown format {}'.format(args.format))
 
 
+def _do_config_proposal_vote(args):
+    """Executes the 'proposal vote' subcommand.  Given a key file, a proposal
+    id and a vote value,, it generates a batch of sawtooth_config transactions
+    in a BatchList instance.  The BatchList is file or submitted to a
+    validator.
+    """
+    pubkey, signing_key = _read_signing_keys(args.key)
+    rest_client = RestClient(args.url)
+
+    proposals = _get_proposals(rest_client)
+
+    proposal = None
+    for candidate in proposals.candidates:
+        if candidate.proposal_id == args.proposal_id:
+            proposal = candidate
```

Comment:
Should this exit the loop after finding a match? Can there ever be more than one match?